src/main.py

```diff
@@ -349,7 +349,20 @@ def play_audio(file, offset=0, overRideVolume=0):
 	global nav_token, p, audioplaying
 	if debug: print("{}Play_Audio Request for:{} {}".format(bcolors.OKBLUE, bcolors.ENDC, file))
 	platform.indicate_playback()
-	i = vlc.Instance('--aout=alsa') # , '--alsa-audio-device=mono', '--file-logging', '--logfile=vlc-log.txt')
+
+	parameters = [
+		# '--alsa-audio-device=mono'
+		# '--file-logging'
+		# '--logfile=vlc-log.txt'
+	]
+
+	if config['sound']['output']:
```

Comment:
Yes. I'd like to have these options in the config mandatory (even when they're empty). Do you think it's a bad idea?